index.js

```diff
@@ -118,7 +118,7 @@ module.exports = function(file, opt) {
         function trykillfn() {
             try {
                 if (worker.kill) {
-                    worker.kill();
+                    worker.kill('SIGKILL');
```

Comment:
@seangarner this is what you was referring in slack. It's not tested. I'll try to setup a test case in dev env if that's the right fix.

Reply:
Yep, this should do it.  You might consider removing the `if (worker.kill)` and `else` parts because they are for a deprecated version of node.  That'd make our change clearer that it's only supported in newer versions of node.

Reply:
https://github.com/dynamicaction/recluster/pull/2/commits/474c73b7c9cc3315683cdb221212c00b715cad56